docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -10,7 +10,8 @@ weight: 600
 The Tempo repository provides [multiple examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) to help you quickly get started using Tempo and distributed tracing data.
 
 Every example has a `docker-compose.yaml` manifest that includes all of the options needed to explore trace data in Grafana, including resource configuration and trace data generation.
-The Tempo examples running with Docker using docker-compose all include a version of Tempo and a storage configuration like Amazon S3 and GCS.
+
+The Tempo examples running with Docker using docker-compose include a version of Tempo and a storage configuration like Amazon S3 and GCS.
```

Comment:
Is this still true? The GCS/S3/Azure examples are deleted in this PR.

Reply:
I wasn't sure how to word that.  It is true that it includes a storage configuration, and one of them has a minio config.  I removed the word "all" here to be less specific about some of the details.

Reply:
Okay I've updated a little.